Cargo.toml

```diff
@@ -45,7 +45,7 @@ bs58 = "0.4"
 rand = "0.8"
 aes = "0.7"
 
-dusk-wallet-core = "0.14.0-rc"
+dusk-wallet-core = "0.14.1-rc"
```

Comment:
Oh damn. I should have reviewed the PR for wallet-core too apparently. You shouldn't have bumped the minor of the version here, but the minor of the `rc`, since it wasn't release yet… To be clear, we had `0.14.0-rc.0` before, you should have add `0.14.0-rc.1`.

(If it was a breaking change it should have been `0.15.0-rc.0`, but I assume is not otherwise we have a major issue and we need to yank the create)

Reply:
You're right
Anyway that release is not a breaking change one